sljit_src/sljitNativeARM_32.c

```diff
@@ -41,6 +41,69 @@ SLJIT_API_FUNC_ATTRIBUTE const char* sljit_get_platform_name(void)
 #endif
 }
 
+#define CPU_FEATURE_NEON	(1UL << 12)
+static unsigned long cpu_feature_list;
+
+#if (defined SLJIT_DETECT_SIMD && SLJIT_DETECT_SIMD)
+#if defined(HAVE_GETAUXVAL) || defined(HAVE_ELF_AUX_INFO)
+#include <sys/auxv.h>
+#endif
+
+#ifdef __NetBSD__
+#include <sys/param.h>
+#include <sys/sysctl.h>
+#endif
+
+static void get_cpu_features(void)
+{
+	if (cpu_feature_list)
+		return;
+
+#if defined(__ARM_ARCH) && __ARM_ARCH == 8
+	/* TODO: confirm if optional with armv9 */
```

Comment:
if true (because NEON is being "replaced" by SVE2, then this might also apply to the 64 bit version.
FWIW some of these code might need to move regardless to a _common file to also apply to T2